src/pages/home/base/mobile-core/api-reference.md

```diff
@@ -448,15 +473,70 @@ You can update the configuration programmatically by passing configuration keys
 
 ## Public classes
 
+### AdobeCallback
+
+The `AdobeCallback` class provides the interface to receive results when the asynchronous APIs perform the requested action.
+
+<TabsBlock orientation="horizontal" slots="heading, content" repeat="1"/>
+
+Android
+
+<Tabs query="platform=android&api=adobe-callback"/>
+
+### AdobeCallbackWithError
+
+The `AdobeCallbackWithError` class provides the interface to receive results or an error when the asynchronous APIs perform the requested action.
+
+When using this class, if the request cannot be completed within the default timeout or an unexpected error occurs, the request is stopped and the fail method is called with the corresponding `AdobeError` or `AEPError`.
+
 <TabsBlock orientation="horizontal" slots="heading, content" repeat="2"/>
 
 Android
 
-<Tabs query="platform=android&api=public-classes"/>
+<Tabs query="platform=android&api=adobe-callback-with-error"/>
 
 iOS
 
-<Tabs query="platform=ios&api=public-classes"/>
+<Tabs query="platform=ios&api=adobe-callback-with-error"/>
+
+### AdobeError
+
+The `AdobeError` class shows the errors that can be passed to an `AdobeCallbackWithError`:
+
+* `UNEXPECTED_ERROR` - An unexpected error occurred.
+* `CALLBACK_TIMEOUT` - The timeout was met.
+* `CALLBACK_NULL` - The provided callback function is null.
+* `EXTENSION_NOT_INITIALIZED` - The extension is not initialized.
+* `SERVER_ERROR` - There was a server error.
+* `NETWORK_ERROR` - There was a network error.
+* `INVALID_REQUEST` - There was an invalid request.
+* `INVALID_RESPONSE` - There was an invalid response.
+
+### AEPError
+
+The `AEPError` enum shows the errors that can be passed to a completion handler callback from any API which uses one:
```

Comment:
Can you mention this class is iOS only? Maybe just `The AEPError enum (iOS) shows ...`?